examples/docker-compose/docker-compose-pro-nginx.yaml

```diff
@@ -5,11 +5,10 @@ services:
     build:
       context: ./botpress
       dockerfile: Dockerfile
-    image: botpress-server
+    image: botpress/server
```

Comment:
I might be mistaken, but I personally find it weird to have both a build and image configuration. Which one is used by default in this case?

Reply:
By default, the image is behind loadd. You can pass the `docker-composer build` argument to build the container from the local image.